implementations/rust/ockam/ockam_command/src/project/addon/configure_okta.rs

```diff
@@ -6,14 +6,15 @@ use clap::builder::NonEmptyStringValueParser;
 use clap::Args;
 use colorful::Colorful;
 use miette::{miette, Context as _, IntoDiagnostic};
-use reqwest::Url;
+use reqwest::Url as RegularUrl;
```

Comment:
Once we introduce a newtype for `Url` I would avoid the exposition of `RegularUrl`.
This means that it would be better to have a `Url::parse` method (delegating to `RegularUrl::parse`).
And when we need to use `host_str`, either go `url.0.host_str()` or implement the `Deref` trait to automatically dereference a `Url` into a `RegularUrl`.  Please have a look at [this post](https://bevyengine.org/news/bevy-0-7/#deref-derefmut-derives) which argues about why this is a good idea for newtypes.